docs/introduction/partials/_getting-started-expo.md

```diff
@@ -2,24 +2,16 @@
 
 :::caution Note
 
-Expo support with Detox is entirely a community driven effort. We have no specific support in Detox for Expo applications (ejected or otherwise).
-
+Expo integration with Detox is entirely a community-driven effort.
+We have no special support for Expo projects, and we do not maintain any Expo-specific code or documentation.
 :::
```

Comment:
Please add a newline before `:::` – this is a lint style.

Reply:
Done :+1: